.github/workflows/issue-creation-tool-versions.yml

```diff
@@ -0,0 +1,65 @@
+---
+name: Evaluate Tool Versions
+
+on:
+  schedule:
+    - cron: 0 0 1 */3 *
+
+permissions: {}
+
+jobs:
+  create-issue:
+    name: Create tool version evaluation issue
+    runs-on: ubuntu-latest
+    permissions:
+      issues: write
+    steps:
+      - run: |
+          if [[ $CLOSE_PREVIOUS == true ]]; then
+            previous_issue_number=$(gh issue list \
+              --label "$LABELS" \
+              --json number \
+              --jq '.[0].number')
+            if [[ -n $previous_issue_number ]]; then
+              gh issue close "$previous_issue_number"
+              gh issue unpin "$previous_issue_number"
+            fi
+          fi
+          new_issue_url=$(gh issue create \
+            --title "$TITLE" \
+            --assignee "$ASSIGNEES" \
+            --label "$LABELS" \
+            --body "$BODY")
+          if [[ $PINNED == true ]]; then
+            gh issue pin "$new_issue_url"
+          fi
+        env:
+          GH_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+          GH_REPO: ${{ github.repository }}
+          TITLE: Evaluate tool versions
+          ASSIGNEES: awesome-embedded-projects
+          LABELS: scheduled,tool-versions
+          BODY: |
+            ## Description
+
+            This issue is automatically created as a gentle reminder to evaluate the tool versions, and base container, for
+            the container image built from this repository. The checklist below describes the steps that should be taken,
+            and checked-off before this issue can be closed. Any decisions about purposefully not updating a tool to a newer
+            version should be logged as issue comment. The general philosophy is to always include the latest tool versions
+            and the latest [LTS](https://ubuntu.com/about/release-cycle) version of Ubuntu.
+
+            Please note that, where possible, tool versions are kept up to date by Dependabot. Below list includes tools that
+            are not under Dependabot control.
+
+            ## Checklist
+
+            - [ ] The image is based on the latest LTS version of Ubuntu
```

Comment:
Isn't this done automatically by dependabot? : https://github.com/philips-software/amp-devcontainer/pull/309

Reply:
That actually is a very good question! I don't know to be fair. Let's see next month 😄.

If Dependabot will update the base image, we can always remove this line again.